source/dataman/HDF5/AMHDF5Group.cpp

```diff
@@ -0,0 +1,98 @@
+#include "AMHDF5Group.h"
+
+#include "util/AMErrorMonitor.h"
+
+AMHDF5Group::AMHDF5Group(const QString &name, QObject *parent)
+	: QObject(parent)
+{
+	name_ = name;
+	groupID_ = 0;
+}
+
+AMHDF5Group::~AMHDF5Group()
+{
+	if(!isOpen())
+		closeHDF5Group();
+}
+
+bool AMHDF5Group::isOpen() const
+{
+	return groupID_ > 0;
+}
+
+bool AMHDF5Group::createHDF5Group(hid_t file)
```

Comment:
Since these are all IDs, lets make that explicit `bool AMHDF5Group::createHDF5Group(hid_t fileId);`